openstack/main.tf

```diff
@@ -338,4 +346,5 @@ module "generate-inventory" {
   gluster_volumetype     = "${var.gluster_volumetype}"
   gluster_extra_disk_dev = "${element(concat(module.glusternode.extra_disk_device, list("")),0)}"
   inventory_template     = "${var.inventory_template}"
+  ingress                = "${var.ingress}"
```

Comment:
Actually, I think it might be even better to move this to the execution of the install-core Ansible-playbook?
provision = {
  "action" = {
    "type"       = "ansible-playbook"
    "playbook"   = "playbooks/install-core.yml"
  }
}

Reply:
e.g.
```
  "action" = {
    "type"      = "ansible-playbook"
    "playbook"  = "playbooks/install-core.yml"
    "extra_vars" = {
      "ingress_controller"       = "traefik"
    }
  }
```